Firestore/core/test/unit/remote/serializer_test.cc

```diff
@@ -954,35 +914,13 @@ TEST_F(SerializerTest, BadStringValue) {
       Status(Error::kErrorDataLoss, "ignored"), bytes);
 }
 
-TEST_F(SerializerTest, BadTimestampValue_TooLarge) {
```

Comment:
Question: do these no longer apply?

Reply:
And similarly in the C++ public API, right?

Reply:
Yes - https://github.com/firebase/firebase-ios-sdk/blob/master/Firestore/core/src/timestamp.cc#L64